src/tests/minicore.md

```diff
@@ -25,6 +25,8 @@ Please note that [`minicore`] is only intended for `core` items, and explicitly
 range of tests.
 </div>
 
+[`minicore`]: https://github.com/rust-lang/rust/blob/master/tests/auxiliary/minicore.rs
```

Comment:
Should we move to the bottom or drop that?

Reply:
Yep, done.